appveyor.yml

```diff
@@ -0,0 +1,23 @@
+branches:  
+  only:
+    - master
+
+cache:
+  - C:\ProgramData\chocolatey\bin -> appveyor.yml
+  - C:\ProgramData\chocolatey\lib -> appveyor.yml
+
+environment:
+  TARGET: x86_64-pc-windows-msvc
+
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
+  - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
+  - SET PATH=%PATH%;C:\Program Files (x86)\Rust\bin
+  - rustc -V
```

Comment:
Why print version?

Reply:
In case of failure, version information is helpful.

Reply:
So how about add it to linux ci too?

Reply:
Travis CI will print it automatically.